wrappers/python/src/python/readdy/api/simulation.py

```diff
@@ -478,8 +479,11 @@ def run(self, n_steps, timestep, show_system=True):
             loop.evaluate_forces(self.evaluate_forces)
             loop.evaluate_topology_reactions(self.evaluate_topology_reactions, timestep)
             loop.use_reaction_scheduler(self.reaction_handler)
-            loop.skin_size = self._skin
-            loop.evaluate_observables(self.evaluate_topology_reactions)
+            loop.evaluate_observables(self.evaluate_observables)
+            if self.integrator == "MdgfrdIntegrator":
+                loop.neighbor_list_distance = max(2. * loop.calculate_max_cutoff(), loop.neighbor_list_distance)
```

Comment:
see previous comment, i think "neighbor list distance" does not fit well into what the parameter does